content/menu.md

```diff
@@ -371,6 +371,14 @@ site_menu:
   - category: Streamlit library / API reference / Connections and databases / Connection base class
     url: /library/api-reference/connections/st.connections.experimentalbaseconnection
     isVersioned: true
+  - category: Streamlit library / API reference / App testing
+    url: /library/api-reference/app-testing
+  - category: Streamlit library / API reference / App testing / st.testing.v1.AppTest
```

Comment:
Could we just do `AppTest` here without the full path, similar to what we do for SQLConnection? But I can see why you might say no.